src/bcf/record.rs

```diff
@@ -1329,6 +1329,9 @@ impl<'a, 'b, B: BorrowMut<Buffer> + Borrow<Buffer> + 'b> Info<'a, B> {
     pub fn string(mut self) -> Result<Option<BufferBacked<'b, Vec<&'b [u8]>, B>>> {
         self.data(htslib::BCF_HT_STR).map(|data| {
             data.map(|ret| {
+                if ret == 0 {
+                    return BufferBacked::new(Vec::new(), self.buffer);
```

Comment:
I'm not sure if this is the behavior we want here. Could consider returning `"."` but then I guess we'd need to modify the buffer